client/components/AttachmentsInputStandalone.tsx

```diff
@@ -33,11 +33,11 @@ export class AttachmentsInputStandalone extends React.PureComponent<IProps, ISta
             <Spacer v gap="16" noWrap>
                 <WithLiveResources
                     resources={[
-                        {resource: 'planning_files', ids: this.props.value},
+                        {resource: 'planning_files', ids: this.props.value ?? []},
                     ]}
                 >
                     {(res) => {
-                        const files: Array<IFile> = res[0]._items;
+                        const files: Array<IFile> = res?.[0]?._items ?? [];
```

Comment:
after https://github.com/superdesk/superdesk-client-core/pull/4762 you can revert this line to how it was